client/src/SearchResult.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import axios from 'axios';
+import ReactPaginate from 'react-paginate';
+import propTypes from 'prop-types';
+import SearchList from './SearchList';
+
+class SearchResult extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      items: [],
+    };
+    this.setInitialPage = this.setInitialPage.bind(this);
+    this.setCurrentPage = this.setCurrentPage.bind(this);
+  }
+
+  setInitialPage() {
+    const { title } = this.props;
+    axios.get(`/books/${title}`, { params: 1 })
+      .then((response) => {
```

Comment:
please add catch blocks to handle errors! if there is an error or timeout in the response, the app could possibly fail to load